lib/kb_Metrics/metricsdb_controller.py

```diff
@@ -472,14 +472,11 @@ def get_jobdata_from_ws_exec_ujs(self, params, token):
 
         exec_apps = self.metrics_dbi.list_exec_apps(params['minTime'], params['maxTime'])
 
-        # subfunc only needs 'app_job_id' and 'app_job_state'
-        exec_apps = [{'app_job_id': exec_app.get('app_job_id'),
-                      'app_job_state': exec_app.get('app_job_state')} for exec_app in exec_apps]
-
         ujs_jobs = self.metrics_dbi.list_ujs_results(
             params['user_ids'], params['minTime'], params['maxTime'])
         ujs_jobs = self.convert_isodate_to_millis(
```

Comment:
you poped 'created', 'started', 'updated'. and 'estcompl' is not used anywhere so should be removed.

Reply:
Did I miss some?  I removed them after the popping.  Instead I am using creation_time, exec_start_time, etc.

Reply:
oh right. your pop is after this.

Reply:
I understand your comment.  Also, I shortened the query for exec_apps so that it only fetch app_job_id and app_job_state.  Once again, if we don' t need to serve job_state, this whole exec_apps can be dropped.